### DISCUSSION TITLE
Input union support

### DISCUSSION THREAD

**Discussion by aisensiy** [Netflix/dgs-framework#817] [Q&A]
https://github.com/graphql/graphql-spec/pull/825

It seems that the contest about input union or similar concepts have last for years. But I just wanna know if there is some way to extend by directive myself in dgs?

**Comment by srinivasankavitha** [comment]
Yes, you can define and add custom directives in graphql-java. The DGS framework allows the same: https://www.graphql-java.com/documentation/sdl-directives
